command.c

```diff
@@ -170,6 +176,7 @@ static int redis_arg3(struct cmd *r) {
 /*
  * Return true, if the redis command accepts 0 or more arguments, otherwise
  * return false
+ * Format: COMMAND key [...]
```

Comment:
What does this mean? Is it `Format: command key [ key ... ]` or `Format: command key [ arg ... ]`?

Reply:
Good, updated to state `arg`